data/RockingSNP.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Souvik Pramanick",
+  "displayStatsPublic": true,
```

Comment:
As of our latest update, we no longer need the `displayStatsPublic` key. So please remove this field from your json. 

```suggestion
```